<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>toSyntaxSetupUI</class>
  <widget class="QWidget" name="toSyntaxSetupUI" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>548</width>
        <height>527</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Form1</string>
    </property>
    <property name="toolTip" >
      <string/>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item row="0" column="1" >
        <widget class="Q3GroupBox" name="Options" >
          <property name="title" >
            <string>Options</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item rowspan="1" row="1" column="0" colspan="3" >
              <widget class="QCheckBox" name="KeywordUpper" >
                <property name="text" >
                  <string>&amp;Keyword upper</string>
                </property>
                <property name="toolTip" >
                  <string>Convert all keywords to uppercase when displayingsyntaxhighlighted text.  Only available for monospaced fonts.</string>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="2" column="0" colspan="3" >
              <widget class="QCheckBox" name="CodeCompletion" >
                <property name="text" >
                  <string>&amp;Code completion</string>
                </property>
                <property name="toolTip" >
                  <string>Activate code completion in SQL editors. Not recommended on slow connections.</string>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="3" column="0" colspan="3" >
              <widget class="QCheckBox" name="CompletionSort" >
                <property name="enabled" >
                  <bool>false</bool>
                </property>
                <property name="text" >
                  <string>&amp;Sort completion rows</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="toolTip" >
                  <string>Sort the available completion rows alphabetically or in the order the are.</string>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="0" column="0" colspan="3" >
              <widget class="QCheckBox" name="SyntaxHighlighting" >
                <property name="text" >
                  <string>S&amp;yntax highlighting</string>
                </property>
                <property name="toolTip" >
                  <string>Enable syntax highlighting, or use simple text editor</string>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="4" column="0" colspan="3" >
              <widget class="QCheckBox" name="AutoIndent" >
                <property name="text" >
                  <string>&amp;Indent same as previous line</string>
                </property>
                <property name="toolTip" >
                  <string>Indent a new line in an editor to the same column as the previous line</string>
                </property>
              </widget>
            </item>
            <item row="7" column="2" >
              <widget class="QPushButton" name="PushButton1" >
                <property name="text" >
                  <string>Choose</string>
                </property>
              </widget>
            </item>
            <item row="7" column="0" >
              <widget class="QLabel" name="ResultExample" >
                <property name="text" >
                  <string>List example</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="6" column="0" >
              <widget class="QLabel" name="CodeExample" >
                <property name="text" >
                  <string>Code example</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="6" column="2" >
              <widget class="QPushButton" name="PushButton1_2" >
                <property name="text" >
                  <string>Choose</string>
                </property>
              </widget>
            </item>
            <item row="5" column="2" >
              <widget class="QPushButton" name="PushButton1_2_2" >
                <property name="text" >
                  <string>Choose</string>
                </property>
              </widget>
            </item>
            <item row="5" column="0" >
              <widget class="QLabel" name="TextExample" >
                <property name="text" >
                  <string>Text example</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="8" column="1" colspan="2" >
              <widget class="QLineEdit" name="Extensions" />
            </item>
            <item row="8" column="0" >
              <widget class="QLabel" name="TextLabel1" >
                <property name="text" >
                  <string>Filename extensions</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="10" column="1" >
              <spacer name="Spacer1" >
                <property name="sizeHint" >
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Qt::Vertical</enum>
                </property>
              </spacer>
            </item>
            <item row="9" column="2" >
              <widget class="QSpinBox" name="TabStop" >
                <property name="minimum" >
                  <number>1</number>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="9" column="0" colspan="2" >
              <widget class="QLabel" name="TextLabel1_2" >
                <property name="text" >
                  <string>Tab stop width</string>
                </property>
                <property name="buddy" >
                  <cstring>TabStop</cstring>
                </property>
                <property name="toolTip" >
                  <string>Number of character width of tabstops</string>
                </property>
                <property name="whatsThis" >
                  <string>Number of character width of tabstops</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="1" row="1" column="0" colspan="2" >
        <widget class="toHighlightedText" name="Example" />
      </item>
      <item row="0" column="0" >
        <widget class="Q3GroupBox" name="GroupBox2" >
          <property name="title" >
            <string>Sy&amp;ntax components</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item row="1" column="0" >
              <widget class="QLabel" name="ExampleColor" >
                <property name="text" >
                  <string/>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="1" >
              <widget class="QPushButton" name="PushButton3" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>4</hsizetype>
                    <vsizetype>0</vsizetype>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>&amp;Pick</string>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="0" column="0" colspan="2" >
              <widget class="Q3ListBox" name="SyntaxComponent" />
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>toHighlightedText</class>
      <extends>QWidget</extends>
      <header location="local" >tohighlightedtext.h</header>
      <sizehint>
        <width>100</width>
        <height>100</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>SyntaxComponent</tabstop>
    <tabstop>PushButton3</tabstop>
    <tabstop>SyntaxHighlighting</tabstop>
    <tabstop>KeywordUpper</tabstop>
    <tabstop>CodeCompletion</tabstop>
    <tabstop>AutoIndent</tabstop>
    <tabstop>PushButton1_2_2</tabstop>
    <tabstop>PushButton1_2</tabstop>
    <tabstop>PushButton1</tabstop>
    <tabstop>Extensions</tabstop>
    <tabstop>TabStop</tabstop>
    <tabstop>Example</tabstop>
    <tabstop>CompletionSort</tabstop>
  </tabstops>
  <images>
    <image name="image0" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
  </images>
  <connections>
    <connection>
      <sender>PushButton1</sender>
      <signal>clicked()</signal>
      <receiver>toSyntaxSetupUI</receiver>
      <slot>selectResultFont()</slot>
    </connection>
    <connection>
      <sender>PushButton1_2</sender>
      <signal>clicked()</signal>
      <receiver>toSyntaxSetupUI</receiver>
      <slot>selectFont()</slot>
    </connection>
    <connection>
      <sender>SyntaxHighlighting</sender>
      <signal>toggled(bool)</signal>
      <receiver>toSyntaxSetupUI</receiver>
      <slot>changeHighlight(bool)</slot>
    </connection>
    <connection>
      <sender>KeywordUpper</sender>
      <signal>toggled(bool)</signal>
      <receiver>toSyntaxSetupUI</receiver>
      <slot>changeUpper(bool)</slot>
    </connection>
    <connection>
      <sender>PushButton3</sender>
      <signal>clicked()</signal>
      <receiver>toSyntaxSetupUI</receiver>
      <slot>selectColor()</slot>
    </connection>
    <connection>
      <sender>SyntaxComponent</sender>
      <signal>selectionChanged(Q3ListBoxItem*)</signal>
      <receiver>toSyntaxSetupUI</receiver>
      <slot>changeLine(Q3ListBoxItem*)</slot>
    </connection>
    <connection>
      <sender>CodeCompletion</sender>
      <signal>toggled(bool)</signal>
      <receiver>CompletionSort</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>SyntaxHighlighting</sender>
      <signal>toggled(bool)</signal>
      <receiver>CodeCompletion</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>PushButton1_2_2</sender>
      <signal>clicked()</signal>
      <receiver>toSyntaxSetupUI</receiver>
      <slot>selectText()</slot>
    </connection>
  </connections>
</ui>
